karma.conf.js

```diff
@@ -16,13 +16,30 @@ module.exports = function (config) {
       require('karma-rollup-preprocessor'),
       require('karma-mocha'),
       require('karma-chrome-launcher'),
-      require('karma-firefox-launcher')
+      require('karma-firefox-launcher'),
+      require('karma-coverage')
     ],
 
+    reporters: ['progress', 'coverage'],
+
     preprocessors: {
       'test/**/*.spec.js': ['rollup']
     },
 
+    coverageReporter: {
+      type: 'text',
+      dir: 'coverage/',
+      file: 'coverage.txt',
+      check: {
+        global: {
```

Comment:
I could reduce these numbers to like 95?

Reply:
@tedconn I'd prefer to have everything at 100 and then use `/* istanbul ignore */` comments in the code for stuff we don't feel we need to cover (e.g. IE-only code).

Reply:
In the short term though, yeah let's just keep it at 91 or something until we can get everything to 100.